src/main/scala/org/wikimedia/analytics/hdfstools/HdfsRsyncConfig.scala

```diff
@@ -105,6 +129,16 @@ case class HdfsRsyncConfig(
      * Parameters validation functions
      */
 
+    def findFirstExistingParent(path: Path, fs: FileSystem): Option[Path] = {
```

Comment:
Do you call this somewhere? I don't see it.

Reply:
Gone - It was a leftover.